src/nn/OperandType.ts

```diff
@@ -0,0 +1,9 @@
+export enum OperandType {
+  'float32' = 'float32',
+  'float16' = 'float16',
+  'int32' = 'int32',
+  'uint32' = 'uint32',
+  'tensor-float32' = 'tensor-float32',
```

Comment:
can you explain the difference between float32 and tensor-float32?

Reply:
Sure. The `float32` is for float32 scalar and `tensor-float32` is for float32 tensor.

Reply:
tensor can also be of scalar type, right?

Reply:
Yes. But the current spec uses the two enums to distinguish scalar or tensor operands.

Reply:
This design originates from [Android NNAPI](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaf06d1affd33f3bc698d0c04eceb23298aee4bc05d71c31e22e39e05470e965447). Feel free to file an issue against it in webnn repo. Thanks!